packages/front-end/components/Experiment/TabbedPage/SetupTabOverview.tsx

```diff
@@ -75,30 +92,76 @@ export default function SetupTabOverview({
             setChecklistItemsRemaining={setChecklistItemsRemaining}
           />
         ) : null}
-
-        <div className="box">
-          <div
-            className="mh-350px fade-mask-vertical-1rem px-4 py-3"
-            style={{ overflowY: "auto" }}
+        <Box className="box" pt="4" pb="2">
+          <Collapsible
+            open={!experiment.description ? true : expandDescription}
+            transitionTime={100}
+            triggerDisabled={!experiment.description}
+            onOpening={() =>
+              localStorage.removeItem(`collapse-${experiment.id}-description`)
+            }
+            onClosing={() =>
+              localStorage.setItem(
+                `collapse-${experiment.id}-description`,
+                "true"
+              )
+            }
+            trigger={
+              <Box
+                as="div"
+                style={{
+                  cursor: `${experiment.description ? "pointer" : "default"}`,
+                }}
+              >
+                <Flex
+                  align="center"
+                  justify="between"
+                  px="5"
+                  pb="2"
+                  className="text-dark"
+                >
+                  <h4 className="m-0">Description</h4>
+                  <Flex align="center">
+                    {canEditExperiment ? (
+                      <button
+                        className={`btn p-0 link-purple ${
+                          experiment.description ? "mr-3" : ""
+                        }`}
+                        onClick={(e) => {
+                          e.stopPropagation();
+                          setShowDescriptionModal(true);
+                        }}
+                      >
+                        Edit
+                      </button>
+                    ) : null}
+                    {experiment.description ? (
+                      <FaAngleRight className="chevron" />
+                    ) : null}
+                  </Flex>
+                </Flex>
+              </Box>
+            }
           >
-            <MarkdownInlineEdit
-              value={experiment.description ?? ""}
-              save={async (description) => {
-                await apiCall(`/experiment/${experiment.id}`, {
-                  method: "POST",
-                  body: JSON.stringify({ description }),
-                });
-                mutate();
-              }}
-              canCreate={canEditExperiment}
-              canEdit={canEditExperiment}
-              label="description"
-              header="Description"
-              headerClassName="h4"
-              containerClassName="mb-1"
-            />
-          </div>
-        </div>
+            {experiment.description ? (
+              <Box
+                as="div"
+                px="5"
+                py="2"
+                maxHeight="491px"
```

Comment:
I kind of feel like this should be larger.... IDK if you have a long-ish description, that's really what this overview page is for.

Reply:
You can also get unlucky and the scroll nature of the window can be really hard to see:

<img width="1354" alt="Screenshot 2025-01-06 at 8 59 27 AM" src="https://github.com/user-attachments/assets/7e34bdc6-eb5f-4d46-aca0-1d53ea0886d1" />

Michael samper had to resolve this on the dimension issues card on the health tab with dark shading:
<img width="495" alt="Screenshot 2025-01-06 at 9 00 10 AM" src="https://github.com/user-attachments/assets/848f186f-9c11-48be-8ba3-7f6f01f23700" />

Reply:
Thanks! I'll leverage the same pattern Michael Samper used

Reply:
I'll share the feedback above re: length with Kari and get her thoughts.